README.md

```diff
@@ -83,10 +83,18 @@ Then set the `YOURAPPLICATION_MODE` variable inside your shell, so the right set
 * 'TESTING': to execute the tests.
 * 'PRODUCTION': to run in production mode.
 
+Set your database. The following example show how to connect to your postgreSQL database `fhe_collector`.
+
+```
+export SQLALCHEMY_DATABASE_URI='postgresql://localhost/fhe_collector'
+```
+
 **Start app**
 
 ```
 export FLASK_APP=fhe.py
+export SQLALCHEMY_DATABASE_URI='postgresql://localhost/fhe_collector'
+export YOURAPPLICATION_MODE='DEVELOPMENT'
 flask run
```

Comment:
Should I be able to simply execute `flask run` after checking out this PR?

Reply:
yes. with the environment variables set as stated in the lines before.